### PR TITLE
docs: add docs for rest position api

### PR DESCRIPTION
**Pull request by GregoryIvo**
This PR adds documentation for the new Rest position API's added in this pr https://github.com/eclipse/kura/pull/4777.

> **Note**: We are using the Conventional Commits convention for our pull request titles. Please take a look at the [PR title format document](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#submitting-the-changes) for the supported [types](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#type) and [scopes](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md#scope).

Brief description of the PR. [e.g. Added `null` check on `object` to avoid `NullPointerException`]

**Related Issue:** This PR fixes/closes {issue number}

**Description of the solution adopted:** A more detailed description of the changes made to solve/close one or more issues. If the PR is simple and easy to understand this section can be skipped

**Screenshots:** If applicable, add screenshots to help explain your solution

**Manual Tests**: Optional description of the tests performed to check correct functioning of changes, useful for an efficient review

**Any side note on the changes made:** Description of any other change that has been made, which is not directly linked to the issue resolution [e.g. Code clean up/Sonar issue resolution]

### CODE REVIEW

File: docs/references/rest-apis/rest-position-api.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	//the ground speed of this position in meters per second.
+	"speed": 1000.0,
+		
+	//the track of this position in degrees as a compass heading.
+	"track": 1000.0
+}
+```
+
+- 400 Bad Request (Malformed Client JSON)
+- 404 Resource Not Found
```
Comment (nicolatimeus) [inline docs/references/rest-apis/rest-position-api.md]: In which cases the 400 and 404 statuses are returned? Same for the other requests

File: docs/references/rest-apis/rest-position-api.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    This API can also be accessed via the RequestHandler with app-id: `POS-V1`.
+
+#### Get Position
+- Method: GET
+- API PATH: `/services/position/v1/position/`
+
+##### Responses
+- 200 OK status
+
+```JSON
```
Comment (nicolatimeus) [inline docs/references/rest-apis/rest-position-api.md:12]: What is the response if we have no fix? Are all fields always guaranteed to be present?

File: docs/references/rest-apis/rest-position-api.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+#### Get Date Time
+- Method: GET
+- API PATH: `/services/position/v1/datetime/`
+
+##### Responses
+- 200 OK status
+
+```JSON
+{
+	"dateTime": "2023-07-19T18:26:38Z"
```
Comment (nicolatimeus) [inline docs/references/rest-apis/rest-position-api.md:50]: What is the response in the case when there is no fix?

File: docs/references/rest-apis/rest-position-api.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+#### Get Is Position Locked
+- Method: GET
+- API PATH: `/services/position/v1/islocked/`
+
+##### Responses
+- 200 OK status
+
+```JSON
+{
+	"localDateTime": "2023-07-19T18:26:38Z"
```
Comment (nicolatimeus) [inline docs/references/rest-apis/rest-position-api.md]: Typo?

File: docs/references/rest-apis/rest-position-api.md (hunk 3 above)
Comment (nicolatimeus) [reply docs/references/rest-apis/rest-position-api.md:50]: We should also document the date format, for example if we are using a standard date format we should reference it.

File: docs/references/rest-apis/rest-position-api.md (hunk 4 above)
Comment (GregoryIvo) [reply docs/references/rest-apis/rest-position-api.md]: A Big typo 😨

File: docs/references/rest-apis/rest-position-api.md (hunk 1 above)
Comment (GregoryIvo) [reply docs/references/rest-apis/rest-position-api.md]: When a position is not locked, a 500 error will be returned with the message "position not locked".

**Review by nicolatimeus — APPROVED** [review]

**Pull request merged**